src/components/MintDialog/pages/Bridge/NotStarted.tsx

```diff
@@ -118,12 +122,14 @@ export const NotStarted: FC<NotStartedProps> = ({
               Bridge now
             </Button>
           )}
-          <Button
-            variant="LIGHT"
-            onClick={() => setPage(ModalPage.CROSS_MINT_FORM)}
-          >
-            Mint with credit card
-          </Button>
+          {crossMintClientId && (
```

Comment:
```suggestion
          { crossMintClientId ? (
          …) : null }
```

can we use a ternary here instaD